setup.py

```diff
@@ -16,9 +16,6 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-with open('requirements.txt') as f:
```

Comment:
we need it to deploy to Pypi... if you want to use the Pipfile instead of the requirements we you can try [this issue](https://github.com/pypa/pipenv/issues/1263)